test/tests/functional/pbs_equiv_classes.py

```diff
@@ -1635,15 +1639,18 @@ def test_multiple_job_preemption_order(self):
         a = {'Resource_List.ncpus': 1}
         J = Job(TEST_USER, attrs=a)
         jid1 = self.server.submit(J)
+        self.server.expect(JOB, {'job_state': 'R'}, id=jid1)
 
         time.sleep(1)
 
         J2 = Job(TEST_USER, attrs=a)
         jid2 = self.server.submit(J2)
+        self.server.expect(JOB, {'job_state': 'R'}, id=jid2)
 
         time.sleep(1)
```

Comment:
and this one too can be removed

Reply:
@anamikau,  I would still want to keep the 1 sec sleep because , sometimes I see that the jobs are started on the MoM at the same time. This affects the preemption order again and test failed. Adding a 1 sec sleep helps.

See logs when I removed the 1 sec sleep. 
Job1:
01/08/2019 11:43:56  L    Evaluating subchunk: ncpus=1
01/08/2019 11:43:56  L    Allocated one subchunk: ncpus=1
01/08/2019 11:43:56  S    enqueuing into workq, state 1 hop 1
01/08/2019 11:43:56  S    Job Queued at request of pbsuser@testdev-01-s11p2.pbspro.com, owner = pbsuser@testdev-01-s11p2.pbspro.com, job name = STDIN, queue = workq
01/08/2019 11:43:56  S    vnode testdev-01-s11p2's parent mom testdev-01-s11p2.pbspro.com:15002 has a pending copy hook or delete hook request
01/08/2019 11:43:56  S    Job Run at request of Scheduler@testdev-01-s11p2.pbspro.com on exec_vnode (testdev-01-s11p2:ncpus=1)
**01/08/2019 11:43:56  L    Job run
01/08/2019 11:43:57  M    Started, pid = 13751**

Job2:
01/08/2019 11:43:57  L    Considering job to run
01/08/2019 11:43:57  L    Evaluating subchunk: ncpus=1
01/08/2019 11:43:57  L    Allocated one subchunk: ncpus=1
01/08/2019 11:43:57  S    enqueuing into workq, state 1 hop 1
01/08/2019 11:43:57  S    Job Queued at request of pbsuser@testdev-01-s11p2.pbspro.com, owner = pbsuser@testdev-01-s11p2.pbspro.com, job name = STDIN, queue = workq
01/08/2019 11:43:57  S    Job Run at request of Scheduler@testdev-01-s11p2.pbspro.com on exec_vnode (testdev-01-s11p2:ncpus=1)
**01/08/2019 11:43:57  L    Job run
01/08/2019 11:43:57  M    Started, pid = 13760**